frontend/src/App.tsx

```diff
@@ -115,20 +115,20 @@ export interface Props {
 
 interface State {
   connectionState: ConnectionState
-  elements: ReportRoot
+  elements: AppRoot
   isFullScreen: boolean
-  reportId: string
-  reportName: string
-  reportHash: string | null
-  reportRunState: ReportRunState
+  sessionId: string
```

Comment:
`sessionId` now has two meanings (we already have a session_id that's used in `ReportSession`). I _think_ this sessionId is just a renamed `Report.report_id`? Which is not really a session ID - it's an ID that's generated per `streamlit run myapp.py` execution, but not per-connected user.

Reply:
That is a good point, I missed that there were two meanings.